tests/serac_linelastic_solver.cpp

```diff
@@ -23,7 +23,7 @@ TEST(elastic_solver, static_solve)
   // Open the mesh
   std::string mesh_file = std::string(SERAC_REPO_DIR) + "/data/meshes/beam-quad.mesh";
 
-  auto pmesh = buildMeshFromFile(mesh_file, 1, 0);
+  auto pmesh = mesh::finalize(*buildMeshFromFile(mesh_file), 1, 0);
```

Comment:
Dereferencing this looks a bit weird for something so user-facing. I think `finalize` should just take a pointer. Thoughts @white238 ?

Reply:
My preference here would be to have the `buildMeshFromFile` etc functions return an `mfem::Mesh` instead of a smart pointer (since they have no need to be on the heap), but for consistency that would require changing all the other signatures, which I was planning on doing when making all the other shapes available through Inlet.